src/cli/mod.rs

```diff
@@ -17,24 +18,23 @@ pub struct ParsecToolApp {
     #[structopt(short = "v", multiple = true, default_value = "0")]
     pub verbosity: u8,
 
-    /// Sets the authentication secret -- will default to no authentication if unspecified.
-    #[structopt(short = "a", long = "auth-secret")]
-    pub auth_secret: Option<String>,
+    /// Sets the application name -- will default to no name if unspecified.
+    /// The app name is used when the service uses direct authentication.
+    #[structopt(short = "a", long = "app_name")]
+    pub app_name: Option<String>,
```

Comment:
Could it, should it, default to a specific name if unspecified, like `"parsec-tool"`? Just to avoid a possible error for clients when direct authentication is used.